requirements.txt

```diff
@@ -2,10 +2,10 @@ arrow>=0.12.1
 hashids>=1.2.0
 petname>=2.2
 tenacity>=5.0.2
-jsonschema@git+https://github.com/Julian/jsonschema@v3.0.0a3#egg=jsonschema
```

Comment:
@mwvaughn I assume we no longer need this specific build pinned; it was breaking 3.7 and 3.8 installs